tortoise/queryset.py

```diff
@@ -536,6 +536,18 @@ def count(self) -> "CountQuery":
             offset=self._offset,
         )
 
+    def exists(self) -> "ExistsQuery":
```

Comment:
Could you add `, *args: Q, **kwargs: Any`
and then call `queryset = self.filter(*args, **kwargs)` and use that queryset instead?

That should allow for a shorter form of `await Desks.exists(owned_by=jon)` whilst still allowing the longer form that you implemented to continue working.

Reply:
I think do filtering in ``exists`` is not comfortable,it should like other methods like ``count()``,``first()`` etc,right?